hphp/runtime/base/stat-cache.cpp

```diff
@@ -937,5 +938,12 @@ std::string StatCache::realpath(const char* path) {
   return s_sc.realpathImpl(path);
 }
 
+void StatCache::clearCache() {
+  if (!RuntimeOption::ServerStatCache) return;
+
+  SimpleLock lock(s_sc.m_lock);
+  s_sc.reset();
```

Comment:
This is still calling .reset(), not .clear()
Would you mind changing it on the PR? If that's inconvenient I can do that locally before landing it.

Reply:
Ahh shoot - I can make that change and re-test here. If you think it's safe, feel free to make that change locally and merge it, and I'll do the same in our branch. Otherwise, happy to test with that change here first. From a static code read, I *think* it's fine.

Reply:
Automated testing takes a long time on my end, so I'm not in a hurry. Please update when you're ready and I'll re-import it again.